packages/jest-runner/test/integration/jest-test-runner.it.spec.ts

```diff
@@ -161,24 +161,6 @@ describe(`${JestTestRunner.name} integration test`, () => {
       assertions.expectSurvived(secondResult);
     });
 
-    it('should only report the first failing test in `killedBy` when disableBail = false', async () => {
```

Comment:
Does this mean that mimicking the bail behavior (`disableBail = false`) for jest no longer works?

Reply:
You're right. I kept the behaviour for this to keep working